shared/domains/chelonia/chelonia.js

```diff
@@ -626,7 +626,7 @@ export default (sbp('sbp/selectors/register', {
               console.warn(`[chelonia] Syncing contract ${channelID} failed: ${err.message}`)
             })
           }
-          options.handlers?.['subscription-succeeded']?.(event)
+          options.handlers?.['subscription-succeeded']?.call(this, event)
```

Comment:
You are exposing Chelonia's internals to the handlers? Why? Is it completely safe to be doing this? 🤔

Reply:
This is exposing the pubsub AFICT, not Chelonia internals.